# tag_msgs
string      epc
int32[]     antID
float64[]     rssi
float64[]   phase
float64[]   timestamp_ros
float64[]   timestamp_pc